socorro/processor/breakpad_transform_rules.py

```diff
@@ -374,6 +380,22 @@ def _execute_external_process(self, command_line, processor_meta):
         return external_command_output, return_code
 
     #--------------------------------------------------------------------------
+    @staticmethod
+    def dot_save(a_mapping, key, value):
```

Comment:
Let's leave it as is then. @twobraids can maybe refactor it some day in the future.